wp-content/themes/inf_theme/functions.php

```diff
@@ -31,7 +31,7 @@
  * @since 3.0.0
  * @package Inf_Theme
  */
-define( 'INF_ASSETS_PUBLIC_URL', get_template_directory_uri() . '/skin/public/' );
+define( 'INF_ASSETS_PUBLIC_PATH', get_template_directory() . '/skin/public/' );
```

Comment:
The `INF_ASSETS_PUBLIC_PATH` should be in the rename script as well, no?

Reply:
it should, could you please add it?

Reply:
Will do :+1: